CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2017 Trail of Bits, Inc.
+# Ceopyright (c) 2017 Trail of Bits, Inc.
```

Comment:
`Copyright(c) 2018`

Reply:
That whole clause should be on top of every source/header file excluding tests?

Reply:
On top of every file.